### PR TITLE
create basic contactus form

### PR DESCRIPTION
**Pull request by yanDavid21**
Prototypical form for contacting LBFE team.

changelist:
added email field
added first and last name fields
added message text area
added button to check for empty fields onClick, throws error if so
added appropriate error message for missing fields

still need:
email regex validation
styling
onClick fetch

### CODE REVIEW

**Review by varunthakkar1 — DISMISSED** [review]

File: lib/components/ContactUsForm.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            isFirstNameMissing={isFirstNameMissing}
+            isLastNameMissing={isLastNameMissing}
+          />
+
+          <FormLabel mt={1} htmlFor="email">
+            Email
+          </FormLabel>
+          <Input
+            id="email"
+            type="email"
```
Comment (rymaju) [inline lib/components/ContactUsForm.tsx:91]: 💬So this comes with email validation so we dont need to do any fancy regex

File: lib/components/ContactUsForm.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            mt={2}
+            colorScheme="blue"
+            onClick={() => {
+              setEmailMissing(emailInput.length === 0);
+              setFirstNameMissing(firstNameInput.length === 0);
+              setLastNameMissing(lastNameInput.length === 0);
+              setMessageMissing(messageInput.length === 0);
+
+              if (emailInput && firstNameInput && lastNameInput && messageInput) {
+                // fetch(`www.example.com`)
```
Comment (rymaju) [inline lib/components/ContactUsForm.tsx:122]: 🔧 Delete this comment

File: lib/components/ContactUsForm.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  Center,
+  Box,
+  Button,
+  Textarea,
+} from '@chakra-ui/react';
+
+interface contactFormProps {}
+
+interface NameFormProps {
+  firstNameInput: string;
```
Comment (rymaju) [inline lib/components/ContactUsForm.tsx:17]: ⛏ Not a huge fan of the "Input" suffix on these

File: lib/components/ContactUsForm.tsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  lastNameInput: string;
+  setFirstInput: any;
```
Comment (rymaju) [inline lib/components/ContactUsForm.tsx:19]: 🔧 any is Not Allowed in this household!

Instead either use the built in set state type from React, or a more readable function type like:
`(string) => void`

File: lib/components/ContactUsForm.tsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  FormErrorMessage,
+  HStack,
+  Input,
+  Center,
+  Box,
+  Button,
+  Textarea,
+} from '@chakra-ui/react';
+
+interface contactFormProps {}
```
Comment (rymaju) [inline lib/components/ContactUsForm.tsx:14]: 🔧 Delete this?

File: lib/components/ContactUsForm.tsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+interface NameFormProps {
+  firstNameInput: string;
+  lastNameInput: string;
+  setFirstInput: any;
+  setLastInput: any;
+  isFirstNameMissing: boolean;
```
Comment (rymaju) [inline lib/components/ContactUsForm.tsx:21]: 💭 I wonder if this calls for a form validation library

File: lib/components/ContactUsForm.tsx [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+          isInvalid={isLastNameMissing}
+        />
+        {isLastNameMissing && <FormErrorMessage>{`Please enter a second name.`}</FormErrorMessage>}
+      </Box>
+    </HStack>
+  );
+};
+
+const ContactForm: React.FC<contactFormProps> = () => {
+  const [emailInput, setEmailInput] = useState('');
```
Comment (rymaju) [inline lib/components/ContactUsForm.tsx:63]: 🔧 We should pass a type to useState if possible
```suggestion
  const [emailInput, setEmailInput] = useState<string>('');
```

**Review by rymaju — CHANGES_REQUESTED** [review]

File: lib/components/ContactUsForm.tsx (hunk 4 above)
Comment (yanDavid21) [reply lib/components/ContactUsForm.tsx:19]: ah I forgot to go back and change it, yep thank you!

File: lib/components/ContactUsForm.tsx (hunk 6 above)
Comment (yanDavid21) [reply lib/components/ContactUsForm.tsx:21]: good question, it was a bit tedious, if more fields are needed and complex logic is needed then yeah, but rn designs just call for an additional subject field

File: lib/components/ContactUsForm.tsx (hunk 5 above)
Comment (yanDavid21) [reply lib/components/ContactUsForm.tsx:14]: yep, agreed, forgot to delete hehe

File: .example.env [hunk 8]
```diff
@@ -1 +0,0 @@
-DATABASE_URL="postgresql://pguser:pgpassword@localhost:port/vsc?schema=public"
```
Comment (rymaju) [inline .example.env:1]: ❓ Why delete this

**Review by rymaju — COMMENTED** [review]
This is good for now! I think in the future

**Pull request closed without merging**